Affinetransform (#6052)
qgis/QGIS-Documentation: docs/user_manual/processing_algs/qgis/vectorgeometry.rst

      :local:
      :depth: 1

.. _qgisaffinetransform:

Affine transform
----------------

> Applies an affine transformation to the geometries from a layer.
> Affine transformations can include translation, scaling and rotation.
> The operations are performed in order: scale, rotation, and translation.
> 
> Z and M values (if present) can also be translated and scaled independently.

Reviewer: ```suggestion

Applies an affine transformation to the layer geometries.
Affine transformations can include translation, scaling and rotation.
The operations are performed in the following order: scale, rotation, and translation.

Z and M values (if present) can be translated and scaled.
```

      :local:
      :depth: 1

.. _qgisaffinetransform:

Affine transform |312|
----------------------

Applies an affine transformation to the layer geometries.
Affine transformations can include translation, scaling and rotation.
The operations are performed in the following order: scale, rotation, and translation.

Z and M values (if present) can be translated and scaled.

.. figure:: img/affinetransform.png
   :align: center

>    Vector layers before (on the left), and after an affine transformation (on the rigth). 

Reviewer: ```suggestion
            Vector point layer (green dots) before (left), and after (rigth) an affine
            transformation (translation). 
```

      :local:
      :depth: 1

.. _qgisaffinetransform:

Affine transform |312|
----------------------

Applies an affine transformation to the layer geometries.
Affine transformations can include translation, scaling and rotation.
The operations are performed in the following order: scale, rotation, and translation.

Z and M values (if present) can be translated and scaled.

.. figure:: img/affinetransform.png
   :align: center

> 
>    Vector point layer (green dots) before (left), and after (rigth) an affine
>     transformation (translation). 

Reviewer: ```suggestion

   Vector point layer (green dots) before (left), and after (rigth) an affine transformation (translation). 
```

>             Vector point layer (green dots) before (left), and after (rigth) an affine
>             transformation (translation). 

Reviewer: ```suggestion
   Vector point layer (green dots) before (left), and after (rigth) an affine
    transformation (translation). 
```
